ipatests/test_webui/data_user.py

```diff
@@ -71,6 +122,8 @@
         ('textbox', 'givenname', 'Name3'),
         ('textbox', 'sn', 'Surname3'),
         ('checkbox', 'noprivate', None),
+        ('password', 'userpassword', 'Supersecret123'),
+        ('password', 'userpassword2', 'Supersecret123'),
```

Comment:
DATA3 are used in method `test_noprivate_nonposix` why this test needs to set a password?